shemaeric/Politico
# [ft-163878372]add aside section to vote page

#### What does this PR do?
This PR adds the aside section that lists the offices as links on voting page
#### Description of Task to be completed?
Add the aside section that has the links of offices and show candidates running for that office
#### How should this be manually tested?
This can be manually tested by cloning this repo and do git checkout 163878372-ft-aside and open the page on web browser to see the changes. 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[ft-163878372](https://www.pivotaltracker.com/n/projects/2240082)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/36806576/52593018-95fa9700-2e50-11e9-8a29-a6f5c5cbcc1f.png)
#### Questions:
N/A
